experimental/oak_baremetal_simple_io/README.md

```diff
@@ -0,0 +1,28 @@
+# Simple I/O Channel
+
+This library provides a simple guest driver that can be used as a communications
+channel between a bare-metal guest VM and the host.
+
+The communication is based on shared memory. There are two separate channels,
+one for input and one for output. Each channel has one buffer in shared memory.
+The guest-physical addresses of the shared pages used for input and output
+buffers are communicated via pre-agreed 32-bit I/O ports. This only has to be
+done once at startup. Since each address is 64 bits long each address will be
+split across 2 ports, the first containing the most significant 4 bytes, and the
+next containing the least significant 4 bytes.
```

Comment:
Should this mention which address is sent first? Eg input channel then output channel?

Reply:
nvm, separate ports